finalfrontier-utils/src/util.rs

```diff
@@ -364,6 +365,11 @@ fn build_with_common_opts<'a, 'b>(name: &str) -> App<'a, 'b> {
                 .takes_value(true)
                 .default_value("6"),
         )
+        .arg(
+            Arg::with_name(NO_SUBWORDS)
+                .long("no_subwords")
```

Comment:
A general remark (does not have to be fixed here). We seem to use underscore in options, but normal dashes are easier to type on normal (US) keyboards. So, I think we should consider changing all options with underscores to normal dashes.

Reply:
While being terrible with braces and brackets, German keyboards make it very easy to write underscores. I'm not opposed to changing this in a follow up PR!